mitodl/mitxpro
# Remove course run expiration dates #76

#### Pre-Flight checklist


#### What are the relevant tickets?
[Ticket](https://trello.com/c/n9Yjb1h1/76-remove-course-expiration-dates)

#### What's this PR do?
Removes code that automatically set the course run expiration date 90 days past the end date. Also adds a migration that nullifies all the previous course run expiration dates.

#### How should this be manually tested?
Run the migration, all existing course runs should now have expiration date set to `None`. Make sure all courses with the empty expiration date have links on the dashboard to edX course. Only courses that have explicitly set expiration date in the past should not show the link.
